src/components/features/ZonesTable.tsx

```diff
@@ -0,0 +1,36 @@
+import DataTable from '@/components/shared/DataTable'
+import { useTranslation } from 'react-i18next'
+import ZonesTableActionsMenu from './ZonesTableActionsMenu'
+
+export default function ZonesTable() {
+  const { t: translate } = useTranslation()
+
+  return (
+    <DataTable
+      hasSearchInput={true}
+      isSortTextField={true}
+      sortLabel={translate('zones.labels.markers')}
+      sortOptionsData={['Гуми', 'Масло']}
+      columnsData={[
+        { key: 'name', title: translate('zones.table.name') },
+        { key: 'markers', title: translate('zones.table.markers') },
+        { key: 'isFinalZone', title: translate('zones.table.isFinalZone') },
+        { key: 'actions', title: translate('zones.table.actions'), minWidth: 50, align: 'right' }
+      ]}
+      rowData={[
+        {
+          name: 'Зона 1',
+          markers: 'Гуми',
```

Comment:
make the markers look like that 
![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.fe/assets/18427030/bbcbde17-59c5-456c-ba58-e62606277e9d)
https://mui.com/material-ui/react-chip/#color-chip

And if there are more than 2 markers, it should have another chip that has `+{markers.length -2}`. When you hover over the markers it should show a tooltip with all the markers like that
![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.fe/assets/18427030/4a7f14b6-9f92-4ca6-81e0-f27d09eb7876)
Where the button in the image is a chip with the text `+{markers.length - 2}`

Think about how you would do such a component and what props you would need.

Reply:
Fixed